src/service/dashboard/dashboardHistory.ts

```diff
@@ -0,0 +1,23 @@
+import { getRepository, MoreThanOrEqual } from 'typeorm'
+import { subDays, startOfToday } from 'date-fns'
+
+import config from 'config'
+import { DashboardEntity } from 'orm'
+
+export async function getDashboardHistory(daysBefore?: number): Promise<DashboardEntity[]> {
+  const whereClause = {
+    chainId: config.CHAIN_ID
+  }
+
+  if (daysBefore) {
+    whereClause['timestamp'] = MoreThanOrEqual(subDays(startOfToday(), daysBefore))
```

Comment:
I found these tests are failing
```
    ✕ Test get account growth count filter (24 ms)
    ✕ Test get tx volumes with count filter (18 ms)
    ✕ Test get block rewards with count filter (19 ms)
```
I can fix it by adding 1 to daysBefore variable. Please confirm.

Reply:
it's failed because data didn't generated in test db due to error in dashboardCollector. Now fixed.